pa_config/rca.conf

```diff
@@ -93,46 +93,5 @@
     "ClusterTemperatureRca"
   ],
   "muted-deciders": [],
-  "muted-actions": [],
```

Comment:
We need to clean up these configs, `muted_actions` and `muted_deciders` are also only needed by master node. Can you create an issue for this?